span.go

```diff
@@ -184,16 +182,7 @@ func StartChildSpan(parent Span, operationName string) Span {
 	})
 }
 
-// InjectSpan is a helper that injects a Span instance into a carrier.
-//
-// See Tracer.Injector() and Injector.InjectSpan().
+// InjectSpan is a trivial helper that injects a Span instance into a carrier.
 func InjectSpan(sp Span, format interface{}, carrier interface{}) error {
```

Comment:
It is borderline... the `InjectSpan` helper does remove the slight stuttering of specifying the Span twice. But I will kill it -- easier to add later.